JoinColony/colonyCDapp
# Add reputation change percentage to completed action table

## Description

Added root hash to the action model, this way we can get the old user reputation, before the smiting/awarding and use that to calculate the percentage the rep changed, like how it works on the form version of the table.

## Testing

- Turn on the reputation monitor, except when you are going through the motion unless you want to accelerate the process.
- Create awarding/smiting actions and motions.
- Check that the percentage shown on the table is the same, or close. It may vary a little bit due to the reputation monitor advancing time and the reputation decaying by the time the block number of the action is mined.

![Screenshot 2024-03-11 at 17 01 41](https://github.com/JoinColony/colonyCDapp/assets/18473896/41d18f2e-40a9-4a0f-a462-0f39d662f0ca)
![Screenshot 2024-03-11 at 17 01 56](https://github.com/JoinColony/colonyCDapp/assets/18473896/cd1940f5-e1b0-494e-920b-7008dda89a58)


Block ingestor PR: https://github.com/JoinColony/block-ingestor/pull/190

Resolves #2033
